docs/source/tutorials/plots_intro.ipynb

```diff
@@ -530,6 +530,7 @@
    "id": "5211f6de-d447-46a8-8652-3c9fd77d7915",
```

Comment:
<p>What do you mean with "we have called properties in which we encode aesthetics"? I don't understand that part.</p><p> </p><p>Also, not sure if some separation in the form of a comma or period is needed in the last sentence (function otherwise)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz-plots/pull/261/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>